en/mdn_plus_privacy.md

```diff
@@ -13,15 +13,15 @@ This privacy notice explains what data MDN Plus collects, shares, and why. We al
 
 ### Information we collect {: #InfoWeCollect }
 
-__Firefox Account information.__ This service requires a Firefox Account, which provides MDN Plus read-only access to your Firefox Account profile information, for example your user ID, display name, avatar, email address, locale, and active product subscriptions. Learn more about [Firefox Account data practices](https://www.mozilla.org/privacy/firefox/#firefox-accounts-join-firefox).
+__Mozilla Account information.__ This service requires a Mozilla Account, which provides MDN Plus read-only access to your Mozilla Account profile information, for example your user ID, display name, avatar, email address, locale, and active product subscriptions. Learn more about [Mozilla Account data practices](https://www.mozilla.org/privacy/firefox/#firefox-accounts-join-firefox).
 
 __MDN Plus Account Information.__ When you use MDN Plus features, we receive the data required to provide those features to you. For instance, if you set saved pages, we receive your saved pages.
```

Comment:
lower case A and I, so the style is consistent with other bullet points: `MDN Plus account information`